source/logging.c

```diff
@@ -144,6 +144,13 @@ int aws_logger_init_standard(
     struct aws_allocator *allocator,
     struct aws_logger_standard_options *options) {
 
+#ifdef ANDROID
+    (void)options;
+    extern int aws_logger_init_logcat(
```

Comment:
Slap the API macro on here, since we deliberately hide symbols otherwise?

Reply:
We don't need the API macro, since this is inside the actual exported API function. Am I missing something?